src/components/common/nextorback/NextOrBack.js

```diff
@@ -7,7 +7,8 @@ const NextOrBack = ({ onClick, rightOrLeft }) => {
   const icon = rightOrLeft === "left" ? faArrowCircleLeft : faArrowCircleRight
   return (
     <div className="form-next">
-      <div className="next-symbol" onClick={onClick}>
+      <span className="next-text">Next Page</span>
+      <div className="next-symbol" title="Next Page" onClick={onClick}>
```

Comment:
Instead of hard coding "Next Page" for both the text in line 10 and the title attribute in line 11 you can just put a prop in there. Then whenever you use the NextOrBack component you can input whatever you want the label text to be 
ex: `<NextorBack labelText={"setup roommates"} />`